pkg/builder/webhook_test.go

```diff
@@ -77,7 +77,7 @@ func runTests(admissionReviewVersion string) {
 		close(stop)
 	})
 
-	It("should scaffold a defaulting webhook if the type implements the Defaulter interface", func() {
+	It("should scaffold a custom defaulting webhook if the type implements the CustomDefaulter interface", func() {
```

Comment:
why didn't you remove this test?

There is already `should scaffold a defaulting webhook with a custom defaulter` below.

Reply:
I think this was an oversight, we can delete this one

Reply:
I think same for `should scaffold a custom validating webhook if the type implements the CustomValidator interface`